src/main/flight/imu.c

```diff
@@ -521,6 +522,9 @@ void imuUpdateAttitude(timeUs_t currentTimeUs)
         acc.accADC[Y] = 0;
         acc.accADC[Z] = 0;
     }
+#ifdef USE_GPS_RESCUE
+    updateGPSRescueState();
```

Comment:
It probably makes more sense to stick this into `gpsUpdate()` - it needs GPS to work anyway, and the GPS task is disabled if no GPS is configured.